tests/profiler/test_profiler.py

```diff
@@ -333,7 +333,6 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
         assert any(f"{local_rank}-validation_step" in f for f in files)
 
 
-@RunIf(special=True)
```

Comment:
Hey @akihironitta!

The random seed is already removed from master. You can still check if this `@RunIf` is still necessary though

Reply:
Removed the check, and there seems no problem :)